include/deal.II/lac/constraint_matrix.templates.h

```diff
@@ -670,7 +682,7 @@ distribute_local_to_global (const Vector<LocalType>      &local_vector,
                 continue;
 
               const ConstraintLine &position_j =
-                lines[lines_cache[calculate_line_index(local_dof_indices[j])]];
+                lines[lines_cache[calculate_line_index(local_dof_indices_row[j])]];
```

Comment:
Since this function is an extension of the old function, I would prefer if the old function (that only takes one set of indices) would simply be re-directed to this one, instead of duplicating 80 lines of code).